test/utils/currency.js

```diff
@@ -108,7 +108,7 @@ describe('the currency utility module', () => {
         'the resulting value', () => {
         expect(cur.convertAndFormatCurrency(500, 'USD', 'PLN', { locale: 'en-US' }))
           .to
-          .equal('PLN2,097.31');
+          .satisfy(val => ['PLN2,097.31', 'PLN 2,097.31'].includes(val));
```

Comment:
This is a little ugly, but on my desktop it adds the space, and on Travis it doesn't, so this is the only way I've been able to get it to pass tests on both.